frontend/dockerfile/docs/reference.md

```diff
@@ -707,6 +708,46 @@ guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practi
 
 The cache for `RUN` instructions can be invalidated by [`ADD`](#add) and [`COPY`](#copy) instructions.
 
+### RUN --device
+
+> [!NOTE]
+> Not yet available in stable syntax, use [`docker/dockerfile:1-labs`](#syntax)
+> version. It also needs BuildKit 0.20.0 or later.
+
+```dockerfile
+RUN --device=name,[required]
+```
+
+`RUN --device` allows build to request [CDI devices](https://github.com/moby/buildkit/blob/master/docs/cdi.md)
```

Comment:
For now I link to our internal docs but in follow-up will be a dedicated page on docs website explaining how to set up a builder with CDI.